packages/devextreme/js/ui/shared/accessibility.js

```diff
@@ -158,7 +159,8 @@ export function restoreFocus(instance) {
     if(!instance.option('useLegacyKeyboardNavigation') && focusedElementInfo) {
         const viewInstance = focusedElementInfo.viewInstance;
         if(viewInstance) {
-            const $activeElements = getActiveAccessibleElements(focusedElementInfo.ariaLabel, viewInstance.element());
+            const escapedAriaLabel = focusedElementInfo.ariaLabel?.replace(/"/g, '\\"');
```

Comment:
Now you do not need this check as it is exists inside the `getActiveAccessibleElements` method